torchani/optim.py

```diff
@@ -65,6 +65,12 @@ def step(self, closure=None):
             for p in group['params']:
                 if p.grad is None:
                     continue
+
+                # Perform stepweight decay
+                # p.data.mul_(1 - group['lr'] * group['weight_decay'])  # AdamW
```

Comment:
Please note that this commit provides implementation identical to Neurochem. The corresponding line has to be uncommented to adapt to the [AdamW implementation](https://arxiv.org/pdf/1711.05101.pdf).